.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Build
```

Comment:
in the maker app, the filename for the "Build" action is [main.yml](https://github.com/code-dot-org/browser/blob/main/.github/workflows/main.yml). i think they should be the same, but i'm not sure which convention makes more sense. maybe this one (build.yml after the job rather than the branch)?

Reply:
Given that we now have a case where we have multiple jobs in one repo, I think it makes more sense to have these file names match the job names. I will update the Maker App job.

Reply:
After a little research I think ci.yml with a descriptive "name" value is the right approach here.